content/stable-rc/index.html

```diff
@@ -10,8 +10,6 @@
 <p>
 LTS Release candidate is the build intended for the upcoming LTS release of Jenkins. <a href="https://wiki.jenkins-ci.org/display/JENKINS/Release+Process">See Wiki for more about what it means.</a>
 </p>
-<p>Upcoming version: <b><?php include "/srv/releases/jenkins/version-stable-rc.txt"?></b></p>
```

Comment:
What about just opening https://github.com/jenkinsci/jenkins/blob/stable-XXX/changelog.html on the new tab? Should be enough as a temporary solution